web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCertificateEventsFacadeServiceImpl.java

```diff
@@ -190,6 +190,20 @@ private CertificateEventTypeDTO getCertificateEventType(EventCode eventCode) {
                 return CertificateEventTypeDTO.EXTENDED;
             case KOPIERATFRAN:
                 return CertificateEventTypeDTO.COPIED_FROM;
+            case NYFRFM:
+                return CertificateEventTypeDTO.INCOMING_MESSAGE;
+            case NYSVFM:
+                return CertificateEventTypeDTO.INCOMING_ANSWER;
+            case HANFRFM:
+                return CertificateEventTypeDTO.INCOMING_MESSAGE_HANDLED;
+            case NYFRFV:
+                return CertificateEventTypeDTO.OUTGOING_MESSAGE;
+            case HANFRFV:
+                return CertificateEventTypeDTO.OUTGOING_MESSAGE_HANDLED;
+            case KOMPLBEGARAN:
+                return CertificateEventTypeDTO.REQUEST_FOR_COMPLEMENT;
+            case PAMINNELSE:
+                return CertificateEventTypeDTO.INCOMING_MESSAGE_REMINDER;
             default:
```

Comment:
Do we need to add Complemented as well? And if so, that should be an event that is decorated with more info, if I'm not mistaken.